setup.py

```diff
@@ -1142,6 +1142,7 @@ def main():
         "networkx",
         "jinja2",
         "fsspec",
+        'mkl>=2021.1.1,<=2021.4.0; platform_system == "Windows" and platform_machine != "ARM64"',
```

Comment:
This feels like a wrong place to add the dependency, as Linux wheels might also depend on MKL if compiled with MKL, aren't they?

Reply:
This was a mistake happened while rebasing. Since in pytorch/main mkl was removed, this change is not there anymore.